web/src/composables/useLogs.ts

```
@@ -4342,6 +4412,721 @@ const useLogs = () => {
+      searchObj.meta.searchApplied = true;
+      searchObj.data.functionError = "";
+      if (
+        !searchObj.data.stream.streamLists?.length ||
+        searchObj.data.stream.selectedStream.length == 0
+      ) {
+        searchObj.loading = false;
+        return;
+      }
+
+      if (
+        isNaN(searchObj.data.datetime.endTime) ||
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)